src/plugins/recordTypes/deaccession/messages.js

```diff
@@ -0,0 +1,32 @@
+import { defineMessages } from 'react-intl';
+
+export default {
+  record: defineMessages({
+    name: {
+      id: 'record.deaccession.name',
+      description: 'The name of the record type',
+      defaultMessage: 'Deaccession',
+    },
+    collectionName: {
+      id: 'record.deaccession.collectionName',
+      description: 'The name of a collection of records of the type.',
+      defaultMessage: 'Deaccessions',
+    },
+  }),
+  panel: defineMessages({
+    info: {
+      id: 'panel.deaccession.info',
+      defaultMessage: 'Deaccession Information',
+    },
+    exit: {
+      id: 'panel.deaccession.exit',
+      defaultMessage: 'Proposed Exit Details',
+    },
+  }),
+  inputTable: defineMessages({
+    exitReason: {
```

Comment:
Is this used?

Reply:
I had only used it when testing the layout. Looks like I forgot to remove it.